diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileStructureRule.kt

```diff
@@ -146,6 +146,7 @@ class FileStructureRule(configRules: List<RulesConfig>) : DiktatRule(
                 copyrightWords.any { blockCommentNode.text.contains(it, ignoreCase = true) }
             }
         var headerKdoc = firstCodeNode.prevSibling { it.elementType == KDOC }
+            ?: importsList?.prevSibling { it.elementType == KDOC }
```

Comment:
Shouldn't `prevSibling`s of `firstCodeNode` be a superset of prevSiblings of `importsList`, since `importsList` is a prevSibling of `firstCodeNode`? Or is it not? Could you please describe a little and add a comment in code.

Reply:
https://github.com/cqfn/diKTat/blob/2f96c8814f4832e075776acfe3e403ca0e17e832/diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileStructureRule.kt#L130-L132

Actually, `firstCodeNode` implies any content, instead of white spaces and comments, so if package directive exists, it would be  `firstCodeNode`, and in this case `prevSibling` won't be the superset of of `prevSiblings` of `importsList`.

However, it were extra actions, so I create additional `if` and add comments